chaossearch/client/operation_read_indexing_state.go

```diff
@@ -0,0 +1,56 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+)
+
+// For documentation see: https://docs.chaossearch.io/reference#bucketmodel
+func (client *Client) ReadIndexingState(ctx context.Context, req *ReadIndexingStateRequest) (*IndexingState, error) {
+	method := "GET"
+
+	responseBody, err := makeGetBucketMetadataRequest(method, client, ctx)
+	if err != nil {
+		return nil, fmt.Errorf("request failed: %s", err)
+	}
+
+	var response ReadBucketMetadataResponse
+	err = client.unmarshalJSONBody(responseBody, response)
+	if err != nil {
+		return nil, fmt.Errorf("failed to unmarshal response: %s", err)
+	}
+
+	if _, ok := response.Metadata[req.ObjectGroupName]; !ok {
+		return nil, fmt.Errorf("response doesn't contain the needed Bucket metadata: %s", req.ObjectGroupName)
+	}
+
+	bucketMetadata := &IndexingState{
+		ObjectGroupName: req.ObjectGroupName,
+		Active: false,
+	}
+
+	if response.Metadata[req.ObjectGroupName].State == "Active" {
+		bucketMetadata.Active = true
+	}
+
+	return bucketMetadata, nil
+}
+
+func makeGetBucketMetadataRequest(method string, client *Client, ctx context.Context) (io.ReadCloser, error) {
+	url := fmt.Sprintf("%s/Bucket/metadata", client.config.URL)
```

Comment:
did you find how is it different then [reading it from the bucket tagging](https://github.com/benzaita/terraform-provider-chaossearch/blob/master/chaossearch/client/operation_read_object_group.go#L108)?